frontend/src/routes/(admin)/settings/server-settings/about/index.tsx

```diff
@@ -16,7 +16,7 @@ export const action = action$(async (data, { request, platform }) => {
 	let success = false
 	try {
 		const response = await handleRequestPost(
-			getDatabase(platform),
+			await getDatabase(platform),
```

Comment:
How did this work in the first place??

Reply:
this page wasn't being exposed so I think it just didn't work 😢 

(that's my bad, I rebased a to main and got the promise based getDatabase but must have missed to update it here 😓 )